src/shared/components/formik-fields/__tests__/EditableLabelField.spec.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { configure, render, screen, cleanup, fireEvent, waitFor } from '@testing-library/react';
+import { Formik, FormikConfig } from 'formik';
+import EditableLabelField from '../EditableLabelField';
+import '@testing-library/jest-dom';
+
+type FormData = {
+  editableLabel: string;
+};
+const Wrapper: React.FC<FormikConfig<FormData>> = ({ children, ...formikConfig }) => (
+  <Formik {...formikConfig}>
+    {(formikProps) => (
+      <form onSubmit={formikProps.handleSubmit}>
+        {children}
+        <input type="submit" value="Submit" />
+      </form>
+    )}
+  </Formik>
+);
+
+const initialLabel: string = 'field value';
+
+const fieldName: string = 'editableLabel';
+
+const initialValues = {
+  editableLabel: initialLabel,
+};
+
+jest.mock('react-i18next', () => ({
+  useTranslation: jest.fn(),
+}));
+
+const mockUseTranslation = useTranslation as jest.Mock;
+
+beforeEach(() => {
+  mockUseTranslation.mockImplementation(() => ({ t: (x) => x }));
+});
+
+afterEach(() => {
+  jest.resetAllMocks();
+  cleanup();
```

Comment:
do we need to explicitly call `cleanup`?

https://testing-library.com/docs/react-testing-library/api/#cleanup

Reply:
we wouldn't need to explicitly call cleanup. removed it from after each block.
Thanks for catching it.